lib/Ed25519KeyPair.js

```diff
@@ -390,6 +451,29 @@ function ed25519SignerFactory(key) {
     };
   }
 
+  // FIXME: determine exact release that included ed25519 signing
```

Comment:
This landed in 12.0.0: https://github.com/nodejs/node/pull/26611/commits/7d0e50dcfef98ca56715adf74678bcaf4aa08796